geometry_msgs/Vector3 position
wam_dmp_controller/RPY orientation
bool accepted
float64 elapsed_time
float64 p2p_traj_duration

